docs/eog/eog-in-practice/cw23/event-project-stage/eps-infrastructure.rst

```diff
@@ -0,0 +1,137 @@
+.. _cw23-eps-infrastructure:
+
+CW23 Infrastructure
+====================
+
+As this was the Institute’s first hybrid CW, we kept the majority of the infrastructure the same from CW21 (see :ref:`cw21-eps-infrastructure`) and CW22 to reduce the number of moving parts and focus on adding in-person elements to our existing online setup.
+
+For a future hybrid CW, looking into an all-encompassing virtual events platform that integrates registration, video conferencing, chat and more (such as Zoom Events) could be a natural next step.
+
+Platforms
```

Comment:
Just a general comment - a wibni (wouldn't it be nice if) - what's the reason for the order of these tools - is it the more important ones first or is it just added in the order we started using them - i.e. is there some meaning here - if so would be useful to tease out. If there is no meaning in the order then perhaps doing this alphabetically would be best.